test/integration/apiserver/tracing/tracing_test.go

```diff
@@ -107,15 +108,18 @@ endpoint: %s`, listener.Addr().String())), os.FileMode(0755)); err != nil {
 
 func containsNodeListSpan(req *traceservice.ExportTraceServiceRequest) bool {
 	for _, resourceSpans := range req.GetResourceSpans() {
-		for _, instrumentationSpans := range resourceSpans.GetInstrumentationLibrarySpans() {
+		for _, instrumentationSpans := range resourceSpans.GetScopeSpans() {
 			for _, span := range instrumentationSpans.GetSpans() {
-				if span.Name != "KubernetesAPI" {
+				if span.Name != "HTTP GET" {
```

Comment:
@dashpole is this a test-only change, or an example of a user-facing change in span names?

Reply:
This is a user-facing change in span names.  There will likely be another change after https://github.com/kubernetes/kubernetes/issues/112059 is addressed.  We can call it out.

Reply:
ack, a change there is in-bounds given the feature is alpha. Can you give @dims language to use in the release note?

Reply:
updated release note